ports/tensorflow-cc/portfile.cmake

```
@@ -75,51 +67,313 @@ set(ENV{TF_NEED_IGNITE} 0)
-else()
-    file(COPY ${CURRENT_BUILDTREES_DIR}/${TARGET_TRIPLET}-rel/bazel-bin/tensorflow/libtensorflow_cc.so.1.14.0 DESTINATION ${CURRENT_PACKAGES_DIR}/lib)
-    file(COPY ${CURRENT_BUILDTREES_DIR}/${TARGET_TRIPLET}-rel/bazel-bin/tensorflow/libtensorflow_framework.so.1.14.0 DESTINATION ${CURRENT_PACKAGES_DIR}/lib)
-    file(COPY ${CURRENT_BUILDTREES_DIR}/${TARGET_TRIPLET}-rel/bazel-bin/tensorflow/libtensorflow_cc.so.1.14.0 DESTINATION ${CURRENT_PACKAGES_DIR}/debug/lib)
-    file(COPY ${CURRENT_BUILDTREES_DIR}/${TARGET_TRIPLET}-rel/bazel-bin/tensorflow/libtensorflow_framework.so.1.14.0 DESTINATION ${CURRENT_PACKAGES_DIR}/debug/lib)
+	message(STATUS "Configuring TensorFlow (${BUILD_TYPE})")
+	tensorflow_try_remove_recurse_wait(${CURRENT_BUILDTREES_DIR}/${TARGET_TRIPLET}-${BUILD_TYPE})
+	file(RENAME ${SOURCE_PATH} ${CURRENT_BUILDTREES_DIR}/${TARGET_TRIPLET}-${BUILD_TYPE})
+	set(SOURCE_PATH "${CURRENT_BUILDTREES_DIR}/${TARGET_TRIPLET}-${BUILD_TYPE}")
+
+	vcpkg_execute_required_process(
+		COMMAND ${PYTHON3} ${SOURCE_PATH}/configure.py --workspace "${CURRENT_BUILDTREES_DIR}/${TARGET_TRIPLET}-${BUILD_TYPE}"
+		WORKING_DIRECTORY ${CURRENT_BUILDTREES_DIR}/${TARGET_TRIPLET}-${BUILD_TYPE}
+		LOGNAME config-${TARGET_TRIPLET}-${BUILD_TYPE}
+	)
+
+	if(DEFINED ENV{BAZEL_CUSTOM_CACERTS})
+		file(APPEND ${CURRENT_BUILDTREES_DIR}/${TARGET_TRIPLET}-${BUILD_TYPE}/.bazelrc "startup --host_jvm_args=-Djavax.net.ssl.trustStore='$ENV{BAZEL_CUSTOM_CACERTS}'\n")
+		message(STATUS "Using custom CA certificate store at: $ENV{BAZEL_CUSTOM_CACERTS}")
+		if(DEFINED ENV{BAZEL_CUSTOM_CACERTS_PASSWORD})
+			file(APPEND ${CURRENT_BUILDTREES_DIR}/${TARGET_TRIPLET}-${BUILD_TYPE}/.bazelrc "startup --host_jvm_args=-Djavax.net.ssl.trustStorePassword='$ENV{BAZEL_CUSTOM_CACERTS_PASSWORD}'\n")
+			message(STATUS "Using supplied custom CA certificate store password.")
+		endif()
+	else()
+		if(DEFINED ENV{HTTPS_PROXY})
+			message(STATUS "You are using HTTPS_PROXY. In case you encounter bazel certificate errors, you might want to set: BAZEL_CUSTOM_CACERTS=/path/to/trust.store (and optionally BAZEL_CUSTOM_CACERTS_PASSWORD), and to enable vcpkg to actually use it: VCPKG_KEEP_ENV_VARS=BAZEL_CUSTOM_CACERTS;BAZEL_CUSTOM_CACERTS_PASSWORD")
+			if(CMAKE_HOST_WIN32)
+				message(STATUS "(For BAZEL_CUSTOM_CACERTS please use forward slashes instead of backslashes on Windows systems.")
+			endif()
+		endif()
+	endif()
+
+	message(STATUS "Warning: Building TensorFlow can take an hour or more.")
+	set(COPTS "")
+	set(CXXOPTS "")
+	set(LINKOPTS "")
+	if(BUILD_TYPE STREQUAL dbg)
+		if(VCPKG_TARGET_IS_WINDOWS)
+			set(BUILD_OPTS "--compilation_mode=dbg --features=fastbuild") # link with /DEBUG:FASTLINK instead of /DEBUG:FULL to avoid .pdb >4GB error
+		else()
+			set(BUILD_OPTS "--compilation_mode=dbg")
+		endif()
+
+		separate_arguments(VCPKG_C_FLAGS)
+		separate_arguments(VCPKG_C_FLAGS_DEBUG)
```

This breaks in the presence of escaped spaces, I think. We probably want:

```suggestion
              # set PLATFORM_COMMAND to "WINDOWS_COMMAND" or "UNIX_COMMAND" depending on platform
		separate_arguments(VCPKG_C_FLAGS ${PLATFORM_COMMAND} "${VCPKG_C_FLAGS}")
		separate_arguments(VCPKG_C_FLAGS_DEBUG ${PLATFORM_COMMAND} "${VCPKG_C_FLAGS_DEBUG}")
```

and fix the rest. Not sure if this is actually the correct thing to do.

---

I agree, pushed a fix.